sdk/deviceupdate/iot-device-update-rest/review/iot-device-update.api.md

```diff
@@ -1373,7 +1361,7 @@ export interface DeviceUpdateGetFiledefaultResponse extends HttpResponse {
 // @public (undocumented)
 export interface DeviceUpdateGetFileHeaderParam {
     // (undocumented)
-    headers: RawHttpHeadersInput & DeviceUpdateGetFileHeaders;
+    headers?: RawHttpHeadersInput & DeviceUpdateGetFileHeaders;
```

Comment:
is this change from typespec? what does the typespec look like?

Reply:
this is generated from swagger, and this should be optional https://github.com/kazrael2119/azure-rest-api-specs/blob/387f63b16ff6f8010b1fffa3860fc44ee72a6be9/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json#L4366